tests/test_series.py

```diff
@@ -791,6 +793,13 @@ def test_iloc_setitem_ndarray() -> None:
     values_s.iloc[indices_u64] = -1
 
 
+def test_types_iter() -> None:
+    s = pd.Series([1, 2, 3], dtype=int)
+    iterable: Iterable[int] = s
+    assert_type(iter(s), Iterator[int])
+    assert_type(next(iter(s)), int)
+
+
 def test_types_to_list() -> None:
     s = pd.Series(["a", "b", "c"], dtype=str)
     assert_type(s.tolist(), List[str])
```

Comment:
can you use `check(assert_type(s.tolist(), List[str]), list, str)` here?

Reply:
I have added this.